pori_python/ipr/content.spec.json

```diff
@@ -99,8 +99,7 @@
             "items": {
                 "properties": {
                     "chromosomeBand": {
-                        "example": "X:p12.2",
-                        "pattern": "^(\\S+:\\S+?)?$",
+                        "example": "p12.2",
```

Comment:
Is there a field for the chromosome name somewhere else?

Reply:
I do not think so.  This is only used as extra gene info.

It seems like standard pattern would be either
p12.2

or without the colon
Xp12.2

Reply:
Eleanor and I are worried that the chromosome name might be usefull somehow. I know we briefly talked about it at the last meeting so sorry if I'm asking an already answered question, but why do we have to remove the chromosome prefix? Below is an example of a copyVariant record:
![image](https://github.com/user-attachments/assets/e55f483b-131e-4f8d-b488-206670fa589d)

Reply:
Change the example to Xp12.2

If you want to keep the chromosome information is fine, but getting 20 thousand error messages about chromosome band regex check, that is wrong, is irritating.

In your example above, the proper chromosomeband should be 2p11.2

https://www.sciencedirect.com/topics/immunology-and-microbiology/chromosome-band#:~:text=Nomenclature%20for%20chromosome%20bands%20includes,short%20arm%2C%20band%2012.3).

Reply:
Could we change the description of 'cna' to something like - Copy Number Absolute, for just straight copy number?

I don't think anyone knows what an alteration ratio would be.
  
                  "cna": {
                        "description": "The copy number alteration (CNA) ratio",

Reply:
> getting 20 thousand error messages about chromosome band regex check, that is wrong, is irritating.

I can imagine!
I'm totally fine with 'Xp12.2'. Maybe the colon was inspired by gkb/hgvs ways of describing variants. If ever needed, I suppose we can adapt downstream scripts to extract chromosome name from it.

Reply:
> Could we change the description of 'cna' to something like - Copy Number Absolute, for just straight copy number?

If it's a better description, I don't see why not.